README.md

```diff
@@ -605,6 +632,12 @@ The `profunctor` method takes two arguments:
 When creating data types which satisfy multiple algebras, authors may choose
 to implement certain methods then derive the remaining methods. Derivations:
 
+  - [`equals`][] may be derived from [`lte`][]:
+
+    ```js
+    function (b) { return this.lte(b) && b.lte(this); }
```

Comment:
For consistency with the other derivations, let's drop the space after `function`.

I'd like to see the parameter named `other` rather than `b` to suggest that the method expects an argument of the same type.